Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -4353,7 +4353,7 @@ final class WorkspaceTests: XCTestCase {
 
         // Pin A to 1.0.0, Checkout B to 1.0.0
         let aPath = workspace.urlForPackage(withName: "A")
-        let aRef = PackageReference(identity: "a", path: aPath)
+        let aRef = PackageReference(identity: "A", path: aPath)
```

Comment:
This is the only test I needed to change. I'm pretty sure this behavior was incidental to the test, rather than a mistake in the implementation, but it'd be great to get some more eyes on this.